### PR TITLE
Modify Events page to Transaction Log View 

### LINKED ISSUE #3311: Moving "Events" list and renaming to "Transactions log"
### Story

When viewing a colony there is currently 3 main navigation links, two of which link to seemingly the same thing from a new users perspective, those being Actions and Events. This is confusing as a new user, especially when you can click on actions but you can’t click on events.

### Description

To help reduce confusion and treat Events as a more advanced feature, the location of events is being moved to a less prominent location, it will be renamed to ‘Transactions Log’ and the design will be simplified to separate it from the look of Actions. Please see the list of tasks below within implementation for a breakdown.

### Design

[Figma Link](https://www.figma.com/file/qQoBbyyDXkuHwoqHIOGjEu/Colony-Simplified?node-id=6434%3A81077)

New events list (Transactions log) view.
events-list-2

Updates to the actions list page.
events-list

### This issue is complete when...

- [ ] The 'Actions' page has been updated with the view 'Transations log' and 'Back to actions' links, depending on the view.
- [ ] The new label for 'Actions' and 'Transactions log' is added, depending on the view.
- [ ] The event item design is simplified as of the spec.
- [ ] The 'Events' main menu item is removed.

### PR DESCRIPTION
**Pull request by dit7ya**
## Description

Changes the Events page to a more compact Transactions Log view.

Note to Reviewers: In my opinion the New Action button does not belong to the transaction logs view - that's why I have omitted it. What do you think?

| Before | After  |
| ------ | --------- |
|Screenshot 2022-04-18 at 18-29-32 Actions Colony - Notun| Screenshot 2022-04-19 at 10-47-34 Actions Colony - imaginality|
|Screenshot 2022-04-18 at 18-29-07 Events Colony - Notun|Screenshot 2022-04-19 at 10-47-06 Transactions Log Colony - imaginality|

Resolves #3311.

### CODE REVIEW

File: src/modules/dashboard/components/ColonyActions/ColonyActions.tsx
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/ColonyActions/ColonyActions.tsx:282]: Is this div here needed?

File: src/modules/dashboard/components/ColonyEvents/ColonyEvents.css
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/ColonyEvents/ColonyEvents.css:7]: A css var is missing here: `--size-small`

File: src/modules/dashboard/components/ColonyEvents/ColonyEvents.tsx
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/ColonyEvents/ColonyEvents.tsx:40]: This id should be prefaced with `dashboard.ColonyEvents.` instead of `dashboard.ColonyActions` right? Same with the id of the message below this one

**Review by ArmandoGraterol — CHANGES_REQUESTED** [review]
Looking good! I noticed some details:

The wallet addresses are being shown in red when in the specs they are grey:
FireShot Capture 569 - Transactions Log - Colony - wonker - localhost
FireShot Capture 570 - Colony Simplified – Figma - www figma com

The dates in the event logs have a bolder weight than what they currently have:
FireShot Capture 571 - Transactions Log - Colony - wonker - localhost
FireShot Capture 572 - Colony Simplified – Figma - www figma com

FireShot Capture 572 - Colony Simplified – Figma - www figma com
FireShot Capture 573 - Transactions Log - Colony - wonker - localhost

I think there should be more space between the action/transaction list and the divisor, like in the specs:
FireShot Capture 576 - Actions - Colony - wonker - localhost

Also, the `Actions` and the `Transactions log` title should be a darker color, according to the specs at least.

**Comment by dit7ya** [comment]
@ArmandoGraterol Thanks for the review.

I fixed all the UI issues. I made the wallet addresses red if there is a username and otherwise grey - which I think is what the spec says.

**Comment by dit7ya** [comment]
@rdig, I already have - see here - https://github.com/JoinColony/colonyDapp/pull/3314#issuecomment-1102079936.

File: src/modules/dashboard/components/ColonyActions/ColonyActions.tsx
Comment (arrenv) [reply src/modules/dashboard/components/ColonyActions/ColonyActions.tsx:301]: I think it is good to have the direct route still. Purely because otherwise it will lead to a lot of broken links (bookmarks, history).

Comment (rdig) [reply src/modules/dashboard/components/ColonyActions/ColonyActions.tsx:301]: I'd argue that nobody in their right mind bookmarked the events page.

Than again, I'm good more or less either way, as long as we have some input from product.

So this can stay as it is for now. :+1:

**Comment by dit7ya** [comment]
@rdig Oops - I had the wrong idea about CSS modules being locally scoped (turns out it is not if you are applying styles on tags like `li`). Fixed it.

File: src/modules/dashboard/components/ColonyEvents/ColonyEventsListItem.css
Comment (dit7ya) [reply src/modules/dashboard/components/ColonyEvents/ColonyEventsListItem.css]: Done.

**Comment by rdig** [comment]
@dit7ya if you rewrite your commits during the review process, or rebase your branch, please let the reviewers know.

It's really annoying having to break the local branches only to realize that something has been rewritten.

You rewrote commit 9f895949b5869ee72a1d99ac5c386e30176b922d as commit e614f04a4acebffc2936ba1f180d0d56966f7682

After the review process started, it's better to just add incremental commits, that way reviewers can see what changed, and it makes it easier to review just that subset of changes _(this is especially helpful with large PRs)_

**Review by rdig — APPROVED** [review]
All good now. Very nice work :100: 

Newest/Oldest sorting doesn't really work correctly, but that's not the scope of this issue.

**Review by chinins — APPROVED** [review]
Works well - nice job!

**Comment by dit7ya** [comment]
@rdig Thanks. That's a really good point I missed. My rationale for interactive rebases for most cases was that it seemed imprudent to pollute commit history with many tiny and non-topical commits. However, I totally understand now the issue with reviewing altered git histories. What do you think about rebasing and cleaning up commit history finally before merging - once all reviews are approved?

File: src/modules/dashboard/components/ColonyActions/ColonyActions.css
Comment (ArmandoGraterol) [inline src/modules/dashboard/components/ColonyActions/ColonyActions.css:11]: CSS var missing here

**Review by ArmandoGraterol — APPROVED** [review]
Excellent work! I just noticed a missing CSS var so I'll approve so you can merge this right away after including that var

**Review by arrenv — APPROVED** [review]
I like it, nice work!

**Comment by rdig** [comment]
Why wasn't this put on QA before merging ?

**Pull request merged**
